Tests/DataCollector/DoctrineDataCollectorTest.php

```diff
@@ -49,6 +49,22 @@ public function testCollectEntities()
         $this->assertCount(2, $entities['default']);
     }
 
+    public function testDoNotCollectEntities() : void
+    {
+        $manager   = $this->createMock('Doctrine\ORM\EntityManager');
+        $config    = $this->createMock('Doctrine\ORM\Configuration');
+        $collector = $this->createCollector(['default' => $manager], false);
+
+        $manager->expects($this->never())
+            ->method('getMetadataFactory');
+        $manager->method('getConfiguration')
+            ->will($this->returnValue($config));
+
+        $collector->collect(new Request(), new Response());
+
+        $this->assertEmpty($collector->getEntities());
```

Comment:
Asserting this doesn't make sense to me, any particular reason to do it? Using `getMappingErrors` would at least make *some* sense

Reply:
if `entities` is empty, it's because we never enter in the if condition, otherwise it would be `['default' => []]`. It's also the condition asked by @stof, to not display anything in the template.

You are right, I will add a test on `getMappingErrors`.

Reply:
done